android/src/main/java/io/ably/flutter/plugin/AblyMethodCallHandler.java

```diff
@@ -147,6 +151,7 @@ private void handleAblyException(@NonNull MethodChannel.Result result, @NonNull
   @Override
   public void onMethodCall(@NonNull MethodCall call, @NonNull MethodChannel.Result rawResult) {
     final MethodChannel.Result result = new MethodResultWrapper(rawResult);
+    System.out.println("Ably Flutter platform method called: " + call.method);
```

Comment:
Should this not go via `Log`?

Reply:
Oh yes, thanks, done in b0c3b535687023cde55f5de889956b3975b7e97a
This should now look like:
```
I/System.out(13771): (VERBOSE): io.ably.flutter.plugin.push.PushMessagingEventHandlers: onMethodCall: Ably Flutter platform method "setRestChannelOptions" invoked.
```

I had removed this line before https://github.com/ably/ably-flutter/pull/228 and re-added it after.